grails-app/views/SDS/index.gsp

```diff
@@ -74,7 +74,7 @@ individual record pages of the sandbox will indicate whether or not sensitivity
     %{--<li> <a href="${request.contextPath}/sensitivity-zones.xml">sensitivity-zones.xml</a></li>--}%
 %{--</ul>--}%
 
-The species that make up the individual components of the SDS can be view via the <a href="http://lists.ala.org.au/public/speciesLists?isSDS=eq:true">list tool</a>.
+The species that make up the individual components of the SDS can be view via the <a href="${grailsApplication.config.list.tool.url}/public/speciesLists?isSDS=eq:true">list tool</a>.
```

Comment:
lists.url seems to be missing from application.yml

Reply:
Added in 5b4ffac97b318e17cc792f018654c097bde50fb8